README.md

```diff
@@ -37,23 +36,21 @@ git clone --recurse-submodules git@github.com:mdenet/educationplatform-docker.gi
 This builds the docker images and starts the platform servers.
 ```
 cd ./educationplatform-docker
-docker compose up
+
+docker compose up --build 
 ```
 
+> Note it may take approximately 10 minutes the first time the platform is built.
 
-## Build and run eclipse-based tool services
 
-This builds the eclipse-based tool services e.g. OCL using maven and starts the services.
-```
-cd ./educationplatform-docker/platformtools/services
-mvn clean 
-mvn install
-./launch.sh
-```
-> Note that this can take some time to complete the first time the services are built.
+## Access the platform
 
-Further details of launching a tool service manually can be found in the platformtools [readme](https://github.com/mdenet/platformtools). 
+Once docker has [started](#build-and-run-the-docker-image) the containers, the platform will be available at http://127.0.0.1:8080
```

Comment:
Worth adding that users will need to use `Ctrl-C` in the terminal running `docker compose up` to stop the platform safely?